src/traces/scatter/attributes.js

```diff
@@ -389,6 +389,39 @@ module.exports = {
             anim: true,
             description: 'Sets the marker opacity.'
         },
+        angle: {
+            valType: 'number',
+            min: -360,
+            max: 360,
```

Comment:
FWIW we have an `angle` valType. It doesn't support arrays (but that wouldn't be hard to add), and rather than a hard min/max it folds any value into [-180,180]. Perhaps worth reusing that?

https://github.com/plotly/plotly.js/blob/8206eb50805b248327573b0ea8d2514c5375b06d/src/lib/coerce.js#L169-L172

Reply:
Good call. Addressed in https://github.com/plotly/plotly.js/pull/6297/commits/9e14777e195a85c31378ea3a14d22c507f5a7ea5.